### ISSUE TITLE
Support user and group values

### ISSUE THREAD

**Issue by baldisos** [sbs20/scanservjs#136]
I'm using the docker-image on a Synology in a non-privileged container for scanning images to my Synology NAS over a network attached scanner. Scanning does work fine, so does ocr'ing and saving. But although i set a UID and GID in the environment variables, the output files are always owned by user "root", hence i can't access it with any user that doesn't have root access.

Is there any way to define the user or UID/GID of the files that get saved?

**Comment by sbs20** [comment]
No, not exactly. It's only really designed to allow access to the files through the web UI.

At the moment the image requires running as the root user. It's possible to change this (and I probably will for general security) but there's still the issue of mapping users and privilege to access the scanner itself.

Is it sufficient for you to specify the group of the process instead?

```
docker run -u :1000 -d -p 8080:8080 -v $HOME/scan-data:/app/data/output --name scanservjs-container --privileged scanservjs-image
```

This runs as root but with the group `1000` so the files are accessible by that group.

**Comment by sbs20** [comment]
CLosed in favour of #177
